packages/lb-annotation/src/core/pointCloud/store/index.ts

```diff
@@ -357,6 +387,25 @@ class PointCloudStore {
     }
   }
 
+  // rerender pointcloud by filtered segment data
+  public updatePointCloudBySegment = (segArr: IPointCloudSegmentation[]) => {
```

Comment:
I think this function belong to `render` ? Maybe it can be moved to `packages/lb-annotation/src/core/pointCloud/render/index.ts` file?

Reply:
This function name need to be changed to something more relevant to filtering.